src/pages/reports/collectionIdSearch.js

```diff
@@ -0,0 +1,81 @@
+import { masterSpecimenIDRequirement } from '../../tubeValidation.js';
+import { finalizeTemplate } from '../finalize.js';
+import fieldToConceptIdMapping from "../../fieldToConceptIdMapping.js";
+
+export const collectionIdSearchScreen = async (auth, route) => {
+    const user = auth.currentUser;
+    if (!user) return;
+    const username = user.displayName ? user.displayName : user.email;
+    appState.setState({ 'username': username });
+    collectionIdSearchScreenTemplate(username);
+};
+
+export const collectionIdSearchScreenTemplate = async (username) => {
+    let template = "";
+    template += reportsNavbar();
+    template += ` 
+              <div id="root root-margin" style="padding-top: 25px;">
+                <h3 style="text-align: center; margin: 1rem 0;">Collection ID Search</h3>
+                <div class="row">
+                    <div class="col-lg">
+                        <div class="row form-row">
+                            <form id="specimenLookupForm" method="POST">
+                                <div class="form-group">
+                                    <label class="search-label">Collection ID</label>
+                                    <input class="form-control" autocomplete="off" required type="text" id="masterSpecimenId" placeholder="Enter/Scan Collection ID"/>
+                                </div>
+                                <div class="form-group">
+                                    <button type="submit" class="btn btn-outline-primary">Search</button>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                </div>
+              </div>
+  `;
+
+    document.getElementById("contentBody").innerHTML = template;
+    document.getElementById("navbarNavAltMarkup").innerHTML = nonUserNavBar(username);
+    activeReportsNavbar();
+    searchSpecimenEvent();
+};
+
+const searchSpecimenEvent = () => {
+    const form = document.getElementById('specimenLookupForm');
+    if (!form) return;
+    form.addEventListener('submit', async e => {
+        e.preventDefault();
+        removeAllErrors();
+        let masterSpecimenId = document.getElementById('masterSpecimenId').value.toUpperCase();
+
+        if(masterSpecimenId.length > masterSpecimenIDRequirement.length) masterSpecimenId = masterSpecimenId.substring(0, masterSpecimenIDRequirement.length);
+
+        if (!masterSpecimenIDRequirement.regExp.test(masterSpecimenId) || masterSpecimenId.length !== masterSpecimenIDRequirement.length) {
+            errorMessage('masterSpecimenId', `Collection ID must be ${masterSpecimenIDRequirement.length} characters long and in CXA123456 format.`, true);
+            return;
+        }
+        showAnimation();
+        const biospecimen = await searchSpecimen(masterSpecimenId, true);
+        if (biospecimen.code !== 200) {
```

Comment:
```suggestion
        if (biospecimen.code !== 200 || Object.keys(biospecimen.data).length === 0) {
```
We're updating backend API to return empty data when code 200, if search returns no result.